docs/2.3/saml.md

```diff
@@ -81,7 +81,11 @@ version: v2
 metadata:
   name: OktaSAML
 spec:
-  acs: https://localhost:3080/v1/webapi/saml/acs
+  # display allows to set the caption of the "login" button
+  # in the Web interface
+  display: "Login with Okta SSO"
+
```

Comment:
`acs is the Assertion Consumer Service URL. This should be the address of the Teleport proxy that your identity provider will communicate with.`

Reply:
oh, nice. I'll add that, thanks.